Unifi commit flow report naming (#164)
se-passau/VaRA-Tool-Suite: README.md

```

### Running BenchBuild experiments

> Second, we change into the benchbuild folder and run an experiment like `CommitReportExperiment` on provided projects, in this case we use `gzip`.

Author: ```suggestion
Second, we change into the benchbuild folder and run an experiment that generates `CommitReports` for provided projects, in this case we use `gzip`.
```

```console
cd $VARA_ROOT/benchbuild

> benchbuild -vv run -E GenerateCommitReport gzip

Reviewer: Why was both called _Report_ before and now there is a distinction between _Experiment_ and _Report_?
What should the naming scheme be in the end?
